cinder/volume/drivers/vmware/vmdk.py

```diff
@@ -2453,3 +2484,46 @@ def revert_to_snapshot(self, context, volume, snapshot):
             LOG.debug("Backing does not exist for volume.", resource=volume)
         else:
             self.volumeops.revert_to_snapshot(backing, snapshot.name)
+
+    def migrate_volume(self, context, volume, host):
+        """Migrate a volume to the specified host.
+
+        If the backing is not created, the dest host will create it.
+        """
+        backing = self.volumeops.get_backing(volume.name, volume.id)
+        dest_host = host['host']
+        # If the backing is not yet created, let the destination host create it
+        # In this case, migration is not necessary
+        if not backing:
+            LOG.info("There is no backing for the volume: %(volume_name)s. "
+                     "Creating it on dest_host %(dest_host)s.",
+                     {'volume_name': volume.name, 'dest_host': dest_host})
+            self._remote_api.create_backing(context, host['host'], volume)
```

Comment:
Why do we have to create the backing if it wasn't there before?

Reply:
The single reason is that, if you perform a `--force-host-copy` migration (which was by default previously since this was not implemented in the driver), you would end up with a backing created, because a `initialize_connection()` would be performed by the cinder host.
Question would be, do we want to preserve that or not?

Reply:
I wouldn't. If the volume is used, `initialize_connection()` would be called anyways.